qanary-component-MT-Python-Helsinki-NLP/README.md

```diff
@@ -54,8 +54,8 @@ docker run -d -p 41062:41062 \
     -e SPRING_BOOT_ADMIN_URL='https://webengineering.ins.hs-anhalt.de:43740' \
     -e SPRING_BOOT_ADMIN_USERNAME='admin' \
     -e SPRING_BOOT_ADMIN_PASSWORD='admin' \
-    -e SERVICE_HOST='http://webengineering.ins.hs-anhalt.de' \
-    -e SERVICE_PORT=41062 \
+    -e SERVER_HOST='http://webengineering.ins.hs-anhalt.de' \
+    -e SERVER_PORT=41062 \
```

Comment:
@Perevalov I suggest updating it to some existing demo server or replace the concrete values with a placeholder.